src/index.html

```diff
@@ -4,9 +4,355 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logo" class="header__logo">
+        <img src="images/logo.png" alt="Moyo_header_Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--first">
+            <a href="#Apple" class="nav__link nav__link--first is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Laptops_&_Computers" class="nav__link"
+            data-qa="nav-hover">Laptops & Computers</a>
```

Comment:
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/09e78b1d-af7a-4382-8d36-9e091a55296c)

Fix in all places

Reply:
Not fixed. Please fix the formatting everywhere